### PR TITLE
No Debian 11 / Rocky 9 in 4.2

### PR DESCRIPTION
**Pull request by Bischoff**
See https://bugzilla.suse.com/show_bug.cgi?id=1212437

### CODE REVIEW

File: jenkins_pipelines/environments/manager-4.2-qe-sle-update [hunk 1]
```diff
@@ -1,18 +1,7 @@
 #!/usr/bin/env groovy
 
 node('sumaform-cucumber-provo') {
-    def minionList = 'sle12sp4_client, sle12sp4_minion, sle12sp4_ssh_minion, ' +
```
Comment (vandabarata) [inline jenkins_pipelines/environments/manager-4.2-qe-sle-update:4]: do we want to remove almost all of them for some specific reason? :grimacing:

File: jenkins_pipelines/environments/manager-4.3-qe-sle-update [hunk 2]
```diff
@@ -1,21 +1,7 @@
 #!/usr/bin/env groovy
 
 node('sumaform-cucumber-provo') {
-    def minionList = 'sle12sp4_client, sle12sp4_minion, sle12sp4_ssh_minion, ' +
```
Comment (vandabarata) [inline jenkins_pipelines/environments/manager-4.3-qe-sle-update:4]: same as above

File: jenkins_pipelines/environments/manager-4.2-qe-sle-update (hunk 1 above)
Comment (Bischoff) [reply jenkins_pipelines/environments/manager-4.2-qe-sle-update:4]: yes, the reason is that SLE updates test suite only has one minion

here too, I have been too lazy to open a separate PR

File: terracumber_config/tf_files/SUSEManager-4.2-build-validation-NUE.tf [hunk 3]
```diff
@@ -599,24 +580,9 @@ module "debian10-minion" {
...
-  product_version    = "4.2-released"
-  name               = "min-debian11"
-  image              = "debian11o"
-  provider_settings = {
-    mac                = "aa:b2:92:42:00:6e"
-    memory             = 4096
-  }
+// Debian 11 is not supported by SUSE Manager 4.2
 
-  server_configuration = {
```
Comment (vandabarata) [inline terracumber_config/tf_files/SUSEManager-4.2-build-validation-NUE.tf:613]: Debian 12 might not be supported but isn't this the _server_ configuration? don't we need this?

File: jenkins_pipelines/environments/manager-4.3-qe-sle-update (hunk 2 above)
Comment (Bischoff) [reply jenkins_pipelines/environments/manager-4.3-qe-sle-update:4]: same as above

File: terracumber_config/tf_files/SUSEManager-4.2-build-validation-NUE.tf (hunk 3 above)
Comment (vandabarata) [reply terracumber_config/tf_files/SUSEManager-4.2-build-validation-NUE.tf:613]: nevermind, this is inside the debian 11 module, my bad

Comment (Bischoff) [reply terracumber_config/tf_files/SUSEManager-4.2-build-validation-NUE.tf:613]: this "server_configuration" is a pointer to the server's configuration used by this minion

I agree this is a terrible name

**Review by vandabarata — APPROVED** [review]

**Review by elariekerboull — APPROVED** [review]

**Review by nodeg — APPROVED** [review]

**Pull request merged**